client/codegen.ts

```diff
@@ -0,0 +1,21 @@
+import type { CodegenConfig } from "@graphql-codegen/cli";
+
+const config: CodegenConfig = {
+  schema: "http://localhost:4000",
```

Comment:
Passer en variable d'env
Si besoin mettre une exeption ts-lint
/* @ts-expect-error: out of .env scope */

Reply:
Profites en pour implémenter la mise en place du appoloprovider et routerprovider dans main
Cela évitera que tout le monde le fasse lors de leur première PR

Reply:
uri schema laissé en dur suivant conversation autour de gestion des vars env dans la racine d'un projet Vite